bundles/org.openhab.binding.omnilink/src/main/resources/OH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,173 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="omnilink"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- OmniLink Controller Bridge -->
+	<bridge-type id="controller">
+		<label>OmniLink Controller</label>
+		<description>An OmniLink controller.</description>
+		<channels>
+			<channel id="sysdate" typeId="sysDate"/>
+			<channel id="enable_disable_beeper" typeId="console_enable_disable_beeper">
+				<label>Enable/Disable All Console Beepers</label>
```

Comment:
Labels are expected to be as short as possible. Guideline is 2-3 words with up to 25 chars. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions